src/createPrevArgSaver.js

```diff
@@ -14,8 +14,14 @@
  *
  * @param arg
  */
+
 function createPrevArgSaver(arg) {
-  // write code here
+  let newArg;
+  return (nextArg) => {
```

Comment:
если аргумент один - скобки не нужны
```suggestion
  return nextArg => {
```